examples/contracts/aggregator-consumer/scripts/consumerValidator.ts

```diff
@@ -0,0 +1,72 @@
+import { ethers, starknet, network } from 'hardhat'
+import { Contract, ContractFactory } from 'ethers'
+import { HttpNetworkConfig, StarknetContract, Account } from 'hardhat/types'
+import fs from 'fs'
+import dotenv from 'dotenv'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
+
+dotenv.config({ path: __dirname + '/.env' })
+const AGGREGATOR_NAME = 'Mock_Aggregator'
+const UPTIME_FEED_NAME =
+  '../../../../contracts/starknet-artifacts/contracts/cairo/ocr2/SequencerUptimeFeed/sequencer_uptime_feed'
+
```

Comment:
Why are these two definitions so vastly different?

In any case, I think we need to change/adapt this whole consumer project, so let's sync offline first.